packaging/standalone/standalone-community/src/main/distribution/text/community/conf/neo4j-http-logging.xml

```diff
@@ -21,9 +21,9 @@
 -->
 <configuration>
   <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
-    <file>data/log/http.log</file>
+    <file>logs/http.log</file>
     <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-      <fileNamePattern>data/log/http.%d{yyyy-MM-dd}.log</fileNamePattern>
+      <fileNamePattern>logs/http.%d{yyyy-MM-dd}.log</fileNamePattern>
```

Comment:
This does the right thing, separate fromthe `http.log` changes. the entire file will disappear at that point, but I figured it was nice to change here in the interest of being complete.